src/main/java/uk/ac/ebi/ampt2d/metadata/rest/controllers/StudyRestController.java

```diff
@@ -171,14 +174,14 @@
             "deprecated, it is not possible to update that study object through PATCH /studies/{id} method, it will " +
             "result in NOT FOUND. This new method (PATCH /studies/{id}/patch) allows that study object to be found and" +
             " updated.")
-    @ApiParam(name = "id", value = "StudyId", type = "string", required = true)
+    @ApiParam(name = "id", value = "Study's id", type = "string", required = true, example = "EGAS0001.1")
```

Comment:
If this is the patch by ID, then the example is not right.

Reply:
I removed it already in the latest code.